napari/_qt/_tests/test_qt_utils.py

```diff
@@ -86,18 +84,6 @@ def test_qbytearray_to_str_and_back(qtbot):
     assert str_to_qbytearray(qbytearray_to_str(qbyte)) == qbyte
 
 
-def test_add_flash_animation(qtbot):
```

Comment:
`add_flash_animation` is difficult to test because it removes `_flash_animation` on a different thread, so we can't be sure it will be there in the test.